Cargo.toml

```diff
@@ -1,17 +1,18 @@
 [package]
-name = "podded"
-version = "0.5.1"
-description = "Zero-copy types for constraint environments"
-authors = ["Metaplex Developers <dev@metaplex.com>"]
-repository = "https://github.com/metaplex-foundation/podded"
-license = "MIT"
-edition = "2021"
-readme = "README.md"
+  authors = ["Metaplex Developers <dev@metaplex.com>"]
+  description = "Zero-copy types for constraint environments"
+  edition = "2021"
+  license = "MIT"
+  name = "podded"
+  readme = "README.md"
+  repository = "https://github.com/metaplex-foundation/podded"
+  version = "0.5.1"
 
 [lib]
-crate-type = ["lib"]
-path = "src/lib.rs"
-test = true
+  crate-type = ["lib"]
+  path = "src/lib.rs"
+  test = true
```

Comment:
Do we need these additional spaces in ` .toml`?

Reply:
No, rogue VS extension, whoops